README.md

```diff
@@ -72,3 +78,7 @@ optional arguments:
   you run the code will be formatted.
 * As a matter of convenience, you may set `$BASE_CODE_DIR` in your environment and run the script
   in the case this is better for your needs.
+* `--skip` directory names must be relative to current directory. For example, if the current directory
+  contains the directory `env/bin`, this entire directory can be skipped with `--skip=env/bin`
+* `--skip` file names can not include the directory names, only file names are allowed. For example, 
+  if there is file name "foo/bar/yo.py", it is skipped by `--skip=yo.py`
```

Comment:
I'm assuming this also means that we can only skip files in the current directory?

Reply:
ah maybe not, reading lower it seems all files within the dir/sub_dirs are fair game